framework/pom.xml

```diff
@@ -32,7 +32,7 @@
       <drill.version>${env.DRILL_VERSION}</drill.version>
       <hadoop.version>2.7.1</hadoop.version>
       <runtimeDepenencies>${env.DRILL_HOME}/conf</runtimeDepenencies>
-      <apache-httpcomponents.version>4.1.1</apache-httpcomponents.version>
+      <apache-httpcomponents.version>4.5.5</apache-httpcomponents.version>
```

Comment:
This was for enabling REST over SSL right? It's good we are bumping it up.

Reply:
yes